src/app/component/user/profile/profile.component.html

```diff
@@ -5,5 +5,17 @@
         vero delectus, ratione error tempore necessitatibus iure aperiam quam cupiditate facere minus perferendis quis.
         Fugiat, omnis!
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/82756220-86068c80-9de1-11ea-99e8-cd0517affb21.png)
Please, add some fixes according to the mockup(font color, size, borders for items and bg for the side block)
Also create one more component and move appropriate logic and markup to this component.

Reply:
Got it. Already fixed.